jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListBasicFiltersPresenter.java

```diff
@@ -192,9 +191,16 @@ protected void addSLAComplianceFilter() {
 
     @Override
     protected void onActiveFilterAdded(ActiveFilterItem activeFilterItem) {
-        if (activeFilterItem.getKey().equals(constants.State()) && activeFilterItem.getValue() instanceof List) {
+        if (activeFilterItem.getKey().equals(constants.State()) &&
+                activeFilterItem.getValue() instanceof List) {
             final List<String> values = (List<String>) activeFilterItem.getValue();
             values.forEach(v -> view.checkSelectFilter(constants.State(), String.valueOf(v)));
         }
+
+        if (activeFilterItem.getKey().equals(COLUMN_ERROR_COUNT) &&
+                activeFilterItem.getValue() instanceof List) {
+            final List<String> values = (List<String>) activeFilterItem.getValue();
+            values.forEach(v -> view.checkSelectFilter(constants.Errors(), (String.valueOf(v).equals("0") || String.valueOf(v) == "0") ? "true" : "false"));
```

Comment:
again equals for string is enough, no need to compare identity aferwards.

Reply:
plus v variable is already a String.

Reply:
when we use -Dfull build compare two strings(that's number like "1"), equals is not working.so I add "==" logic it. For my local test(gwt run), must be use equals. I know it's a little weird.